BaragonService/src/main/java/com/hubspot/baragon/service/managers/ElbManager.java

```diff
@@ -162,6 +169,7 @@ private void registerNewInstances(List<LoadBalancerDescription> elbs, BaragonGro
           LOG.info(String.format("Registered instances %s with ELB %s", request.getInstances(), request.getLoadBalancerName()));
         } catch (AmazonClientException e) {
           LOG.error("Could not register %s with elb %s due to error %s", request.getInstances(), request.getLoadBalancerName(), e);
+          exceptionNotifier.notify(e, ImmutableMap.of("elb", request.getLoadBalancerName(), "toAdd", request.getInstances().toString()));
```

Comment:
Kieran attempted the logback thing a long time ago, and it never really worked. But more than that, explicitly triggering the notification allows us to add extra information, and control how the messages are deduped in Sentry.